aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1,54 +1,28 @@
 import { Anchor, Box, Card, Text } from 'grommet';
 import { StatusCritical, StatusInfo } from 'grommet-icons';
 
-Messaging within HPE applications should be informative, concise, and trustworthy.
-It should be delivered in active, positive, objective, conversational
-American English and written from a third-person [point of view](#point-of-view).
-Executed properly, messaging within HPE applications should leave users
-feeling and perceiving HPE and its products to be intelligent, experienced,
-inclusive, and trustworthy.
+The HPE Design System employs text-based content that cultivates an exceptional user experience.
+<!-- The above sentence will be the leading line and moved to the Voice and Tone card in Foundations once approved -->
 
-## Messaging Voice, Tone, and Content
+The user experience is built upon our content. Every word we write is a carefully-considered concept that voices our values: 
 
-Many definitions for tone and voice exist, however, reading through the
-various definitions, a common picture emerges:
+- **Open**: We foster inclusive experiences that are personal, accessible, and transparent.
+- **Courageous**: We empower users with decisive and experienced guidance.
+- **Inspired**: We share our drive of 'yes we can' with users.
```

Comment:
I like the connection to the HPE values! It might be good to look for more opportunities to include others.